nengo/tests/test_simulator.py

```diff
@@ -53,6 +53,19 @@ def test_probedict():
     assert np.all(probedict.get("list") == np.asarray(raw.get("list")))
 
 
+def test_probedict_with_repeated_simulator_runs(RefSimulator):
+    with nengo.Network() as model:
+        ens = nengo.Ensemble(10, 1)
+        p = nengo.Probe(ens)
+
+    dt = 0.001
+    with nengo.Simulator(model, dt=dt) as sim:
```

Comment:
I though of that when writing the function signature, but then I forgot when writing the actual code. :fearful: